en: update lightning deployment using S3 IAM (#86)
pingcap/docs-tidb-operator: en/restore-data-using-tidb-lightning.md

              service_account_credentials = <service-account-json-file-content>
            ```

        * If you grant permissions by associating AWS S3 IAM with Pod or with ServiceAccount, you can ignore `s3.access_key_id` and `s3.secret_access_key`:

            {{< copyable "" >}}
    
            ```yaml
            apiVersion: v1
            kind: Secret
            metadata:
              name: cloud-storage-secret
            type: Opaque
            stringData:
              rclone.conf: |
              [s3]
              type = s3
              provider = AWS
              env_auth = true
              access_key_id =
              secret_access_key =
              region = us-east-1
            ```

            Fill in the placeholders with your configurations and save it as `secret.yaml`, and then create the `Secret` via `kubectl apply -f secret.yaml -n <namespace>`.

    3. Configure the `dataSource.remote.storageClassName` to an existing storage class in the Kubernetes cluster.

### Deploy

> The method of deploying TiDB Lightning varies with different methods of granting permissions and different storages.

Reviewer: ```suggestion
The method of deploying TiDB Lightning varies with different methods of granting permissions and with different storages.
```

    2. Create the IAM role:

        [Create an IAM role](https://docs.aws.amazon.com/eks/latest/userguide/create-service-account-iam-policy-and-role.html). Grant the `AmazonS3FullAccess` permission to the role, and edit `Trust relationships` of the role.

    3. Associate IAM with the ServiceAccount resources:

        {{< copyable "shell-regular" >}}

        ```shell
        kubectl annotate sa <servie-account> -n eks.amazonaws.com/role-arn=arn:aws:iam::123456789012:role/user
        ```

    4. Deploy TiDB Lightning:

        {{< copyable "shell-regular" >}}

        ```shell
        helm install pingcap/tidb-lightning --name=<tidb-lightning-release-name> --namespace=<namespace> --set-string failFast=true,serviceAccount=<servie-account> -f tidb-lightning-values.yaml --version=<chart-version>
        ```

        > **Note:**
        >
        > `arn:aws:iam::123456789012:role/user` is the IAM role created in Step 1.
        > `<service-account>` is the ServiceAccount used by TiDB Lightning. The default value is `default`.

> When TiDB Lightning fails to restore data, it cannot simply be restarted. **Manual intervention** is required. So the TiDB Lightning's `Job` restart policy is set to `Never`.

Reviewer: ```suggestion
When TiDB Lightning fails to restore data, you cannot simply restart it. **Manual intervention** is required. So the TiDB Lightning's `Job` restart policy is set to `Never`.
```
